packages/web/docs/src/app/case-studies/more-stories-section.tsx

```diff
@@ -0,0 +1,63 @@
+import { cn, Heading } from '@theguild/components';
+import { getPageMap } from '@theguild/components/server';
+import { CaseStudyCard } from './case-study-card';
+import { CaseStudyFrontmatter } from './case-study-frontmatter';
+import { companyLogos } from './company-logos';
+
+// TODO
+const CURRENT_CASE_STUDY_NAME = 'sound-xyz';
```

Comment:
This component is irrelevant until we have 3 case studies, and I hope @dimaMachina will advise me some neat trick here later.